packages/core/src/input/input-file-resolver.ts

```diff
@@ -119,12 +119,12 @@ export class InputFileResolver {
   }
 
   private async expandPattern(globbingExpression: string, logAboutUselessPatterns: boolean): Promise<string[]> {
-    if (RegExp(/(:\d+){4}/).exec(globbingExpression)) {
+    if (/(:\d+){4}$/.exec(globbingExpression)) {
```

Comment:
is this change intentional?

Reply:
Yup. The specific mutation definition should only occur at the end of the filename/glob expression